iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/HelmProcessor.java

```diff
@@ -86,6 +87,12 @@ String processHelmTemplate(String path, String source, HelmInputFileContext inpu
   }
 
   static Map<String, String> validateAndReadFiles(HelmInputFileContext inputFileContext) {
+    var lineSeparators = List.of("\n", "\r\n");
+    if (lineSeparators.stream().anyMatch(sep -> inputFileContext.inputFile.filename().contains(sep) ||
+      inputFileContext.getAdditionalFiles().keySet().stream().anyMatch(f -> f.contains(sep)))) {
```

Comment:
For me if `inputFileContext.inputFile` contains new line we should throw exception (as it is here)
and if Additional File contains new line we should exclude it and try evaluating. Of course there should also debug log.

What if a directory name contains a new line? We should also cover it!

Reply:
> What if a directory name contains a new line?

We only pass file names to the Go executable, so this should be more or less fine